pages/_app.js

```diff
@@ -1,22 +1,29 @@
 import Layout from '../components/layout'
 import Head from 'next/head'
 import React from 'react'
+import { NextAdapter } from 'next-query-params'
+import { QueryParamProvider } from 'use-query-params'
 
 export default function CourseCatalog({ Component, pageProps }) {
   return (
     <React.Fragment>
-      <Layout>
-        <Head>
-          <title>Course Catalog - Tulsa Public Schools</title>
-          <meta name="description" content="Course Catalog for Tulsa Public Schools" />
-          <link
-            rel="icon"
-            href="/images/tps-logo-color.svg"
-            type="image/svg+xml"
-          />
-        </Head>
-        <Component {...pageProps} />
-      </Layout>
+      <QueryParamProvider adapter={NextAdapter}>
```

Comment:
Getting this weird flickering text bug now:
![flickering text](https://user-images.githubusercontent.com/33558504/212928961-dcc77e3c-7a6b-4046-858d-ff26317e99b7.gif)